src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicApi.cs

```diff
@@ -19,6 +20,9 @@ internal sealed unsafe partial class MsQuicApi
 
     private static readonly Version MsQuicVersion = new Version(2, 1);
```

Comment:
I got confused by this naming.  This isn't the version of msquic that's being used, but rather than minimum supported version, yes?  It should probably be named accordingly, e.g. MinimumSupportedMsQuicVersion.